indexer/categories_holder.cpp

```diff
@@ -203,6 +203,8 @@ void CategoriesHolder::AddCategory(Category & cat, vector<uint32_t> & types)
       auto const locale = synonym.m_locale;
       ASSERT_NOT_EQUAL(locale, kUnsupportedLocaleCode, ());
 
+      auto const localePrefix = String(1, static_cast<strings::UniChar>(locale));
```

Comment:
Do we really need it? The UniString constructor mimics std::string constructor: ```String(numChars, ch)```.